mne/commands/mne_freeview_bem_surfaces.py

```diff
@@ -84,6 +84,7 @@ def run():
 
     subject = os.environ.get('SUBJECT')
     subjects_dir = get_subjects_dir()
+    subjects_dir = None if subjects_dir is None else str(subjects_dir)
```

Comment:
this seems rather odd, I'd suggest to simply check for `not None` and do the assignment in the `if` block

Reply:
You prefer this syntax:

```
subjects_dir = get_subjects_dir()
if subjects_dir is not None:
    subjects_dir = str(subjects_dir)
```

There is also the reversed one-liner condition, which is a bit more natural:

```
subjects_dir = get_subjects_dir()
subjects_dir = str(subjects_dir) if subjects_dir is not None else None
```

Reply:
> So, you prefer this syntax?
> 
> subjects_dir = get_subjects_dir()
> if subjects_dir is not None:
>     subjects_dir = str(subjects_dir)
> 

Yes, exactly

Because it's clear that it will only alter the value of `subjects_dir` in **one** specific situation

Reply:
Done